Clients/GUI/Pages/UserTransactions.razor

```diff
@@ -0,0 +1,81 @@
+﻿@page "/transactions"
```

Comment:
Change the design so it's under menu **UserInfo  Transactions**, so the choice does not disappear after getting to transactions

Reply:
Transactions moved in UserInfo